packages/core/src/_test/setup.ts

```diff
@@ -137,11 +137,8 @@ const defaultDatabaseServiceSetup: DatabaseServiceSetup = {
   indexing: "historical",
 };
 
-vi.mock("@/generated", async () => {
-  return {
-    instanceId: "1234",
-  };
-});
+// @ts-ignore
+globalThis.INSTANCE_ID = "1234";
```

Comment:
I think we should use something like `__PONDER_INSTANCE_ID` to reduce the chance of a collision. See [Next.js example](https://github.com/vercel/next.js/blob/canary/packages/next/src/server/setup-http-agent-env.ts).